### PR TITLE
[TASK] i18n for calcium plugin (#692)

### PR DESCRIPTION
**Pull request by happibum**

### CODE REVIEW

File: apps/ide/src/plugins/webida.editor.code-editor.content-assist.calcium/nls/zh-cn/resource.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+define({
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor.content-assist.calcium/nls/zh-cn/resource.js]: This file is too, I think it's better to remove.
Thanks.

**Comment by happibum** [comment]
@hwshim : Thank you for the comment. I removed the zh-cn resource.

File: apps/ide/src/plugins/webida.editor.code-editor.content-assist.calcium/calcium-server.js [hunk 2]
```diff
@@ -22,70 +22,73 @@
...
+) {
+    'use strict';
 
-        var logger = new Logger();
-        logger.off();
+    var logger = new Logger();
+    logger.off();
 
-        logger.info('CREATING SEVER');
+    logger.info('CREATING SEVER');
```
Comment (hwshim) [inline apps/ide/src/plugins/webida.editor.code-editor.content-assist.calcium/calcium-server.js]: Mispelt SEVER

Comment (happibum) [inline apps/ide/src/plugins/webida.editor.code-editor.content-assist.calcium/calcium-server.js]: Sorry to bother. Fixed it. :)

**Pull request merged**